tests/common/snappi_tests/common_helpers.py

```diff
@@ -811,10 +811,19 @@ def enable_packet_aging(duthost, asic_value=None):
         duthost.command("docker exec syncd python /packets_aging.py enable")
         duthost.command("docker exec syncd rm -rf /packets_aging.py")
     elif "platform_asic" in duthost.facts and duthost.facts["platform_asic"] == "broadcom-dnx":
-        try:
-            duthost.shell('bcmcmd -n {} "BCMSAI credit-watchdog enable"'.format(asic_value))
-        except Exception:
-            duthost.shell('bcmcmd -n {} "BCMSAI credit-watchdog enable"'.format(asic_value[-1]))
+        # if asic_value is present, enable packet aging for specific asic_value.
+        if (asic_value):
+            try:
+                duthost.shell('bcmcmd -n {} "BCMSAI credit-watchdog enable"'.format(asic_value))
+            except Exception:
+                duthost.shell('bcmcmd -n {} "BCMSAI credit-watchdog enable"'.format(asic_value[-1]))
```

Comment:
is asic_value a list ? in case of exception, it may look for last element in the list and may fail if only one is passed ?

Reply:
I think the asic_value can be passed as list or string. This piece of code was added as part of PR #7849 . IMO, the exception just handles the case, where asic_value is passed as list. It would be if the reviewers/author of the above PR can help in understanding.

The piece of code, I have added, just handles situation, where asic_value is not passed at all.